web/gridt_server/tests/base_test.py

```diff
@@ -1,172 +0,0 @@
-import sys
-import logging
-import lorem
-import random
-
-from unittest import TestCase
-
-from gridt_server.app import create_app
-from gridt_server.db import db
-
-from gridt_server.models.movement import Movement
-from gridt_server.models.user import User
-from gridt_server.models.signal import Signal
-
-from sqlalchemy import create_engine
-from gridt.db import Base, Session
-
-
-# LoggedTestCase and LogThisTestCase accredited to:
-# https://stackoverflow.com/a/15969985/1051438
-class LogThisTestCase(type):
-    def __new__(cls, name, bases, dct):
-        # if the TestCase already provides setUp, wrap it
-        if "setUp" in dct:
-            setUp = dct["setUp"]
-        else:
-            setUp = lambda self: None  # noqa: E731
-            print("creating setUp...")
-
-        def wrappedSetUp(self):
-            # for hdlr in self.logger.handlers:
-            #    self.logger.removeHandler(hdlr)
-            self.hdlr = logging.StreamHandler(sys.stdout)
-            self.logger.addHandler(self.hdlr)
-            setUp(self)
-
-        dct["setUp"] = wrappedSetUp
-
-        # same for tearDown
-        if "tearDown" in dct:
-            tearDown = dct["tearDown"]
-        else:
-            tearDown = lambda self: None  # noqa: E731
-
-        def wrappedTearDown(self):
-            tearDown(self)
-            self.logger.removeHandler(self.hdlr)
-
-        dct["tearDown"] = wrappedTearDown
-
-        # return the class instance with the replaced setUp/tearDown
-        return type.__new__(cls, name, bases, dct)
-
-
-class LoggedTestCase(TestCase):
-    __metaclass__ = LogThisTestCase
-    logger = logging.getLogger()
-    # Uncomment below to enable logging while testing
-    # logger.setLevel(logging.DEBUG)
-
-
-class BaseTest(LoggedTestCase):
-    def setUp(self):
-        app = create_app(overwrite_conf="conf/test.conf")
-
-        # Make sure db exists
-        with app.app_context():
-            db.init_app(app)
-            db.create_all()
-        # Get a test client
-        self.app = app
-        self.client = app.test_client()
-        self.app_context = app.app_context
-
-        self.users = []
-        self.movements = []
-
-        engine = create_engine(app.config["SQLALCHEMY_DATABASE_URI"])
-        Session.configure(bind=engine)
-        Base.metadata.create_all(engine)
-
-    def tearDown(self):
```

Comment:
All functions from here down rely on the database / models. They can no longer be used.